### PR TITLE
Depth cameras

### PR DESCRIPTION
**Pull request by SijmenHuizenga**
This pr adds depth camera's. Read the docs on a description of how they work.

### CODE REVIEW

**Comment by SijmenHuizenga** [comment]
https://www.youtube.com/watch?v=CmHY8ng7UgM

**Comment by SijmenHuizenga** [comment]
force pushed to rebase on master

**Comment by SijmenHuizenga** [comment]
https://www.youtube.com/watch?v=T0m2-ZLvZU0

File: ros/src/fsds_ros_bridge/src/fsds_ros_bridge_camera.cpp [hunk 1]
```diff
@@ -96,6 +96,30 @@ cv::Mat manual_decode_depth(const ImageResponse& img_response)
     return mat;
 }
 
+float roundUp(float numToRound, float multiple) 
+{
+    assert(multiple);
+    return ((numToRound + multiple - 1) / multiple) * multiple;
+}
```
Comment (Plavit) [inline ros/src/fsds_ros_bridge/src/fsds_ros_bridge_camera.cpp:103]: I am slightly confused about the rounding - how exactly does it work? Can you provide an example?

Comment (SijmenHuizenga) [reply ros/src/fsds_ros_bridge/src/fsds_ros_bridge_camera.cpp:103]: copied it from here ;) https://stackoverflow.com/a/9194117/2328729

Comment (SijmenHuizenga) [reply ros/src/fsds_ros_bridge/src/fsds_ros_bridge_camera.cpp:103]: it rounds up to the nearest multiple of another number

**Comment by SijmenHuizenga** [comment]
@Plavit Your ideas for further development sounds good! Would you like to create a new issues for this so that it doesn't get lost in pr history? :D

**Comment by Plavit** [comment]
@SijmenHuizenga 
> @Plavit Your ideas for further development sounds good! Would you like to create a new issues for this so that it doesn't get lost in pr history? :D

Hmm, good point - may I suggest just sticking to using the depth camera issue we were using up till now? It is already classified as enhancement if I recall correctly...

**Comment by SijmenHuizenga** [comment]
Let's start with a clean slate in a new ticket and keep the #153 closed. I will mark it as enhancement again :)

**Comment by Plavit** [comment]
@SijmenHuizenga Done: #187

**Pull request merged**